ew/static/cfg.py

```diff
@@ -1734,6 +1744,12 @@
 
 
 crime_status = {
+-5000:"{they} have done more to fight crime than perhaps anyone else. Forget juvieman and forget Mr. C, There is a new sherrif in town.",
+-1000:"{they} are a one man police force.",
+-100:"{they} have contributed towards the peace and prosperity of this city.",
```

Comment:
say NLACakaNM instead of "this city". We gotta integrate the worldbuilding!!!!!!!